### ISSUE TITLE
[Feature]different type of copycat connect

### ISSUE THREAD

**Issue by nickeh913678** [copycats-plus/copycats#158]
please make the different type of copycat connect
![Snipaste_2024-09-08_17-57-01](https://github.com/user-attachments/assets/d85e5523-d130-4eca-97c8-823bbdca9a62)

**Comment by hlysine** [comment]
I don't think we should add a lot of shape connections, because eventually people will ask for versions that don't connect. For the cases above, you can just build the connecting parts with copycat bytes.

**Comment by gfififid** [comment]
> For the cases above, you can just build the connecting parts with copycat bytes.

No, you cant. Bytes can't be placed in the same block with slabs, stairs, byte panel, etc. And that, I suppose, what is asked(And I hope this will be added someday)

**Comment by theTaeco** [comment]
> > For the cases above, you can just build the connecting parts with copycat bytes.
> 
> No, you cant. Bytes can't be placed in the same block with slabs, stairs, byte panel, etc. And that, I suppose, what is asked(And I hope this will be added someday)

you could build the whole thing out of bytes..

**Comment by gfififid** [comment]
> > > For the cases above, you can just build the connecting parts with copycat bytes.
> > 
> > No, you cant. Bytes can't be placed in the same block with slabs, stairs, byte panel, etc. And that, I suppose, what is asked(And I hope this will be added someday)
> 
> you could build the whole thing out of bytes..

Bytes do not provide the same texture as slabs or full blocks. It would be wonderful if i can set 4 bytes act like a slab, but i can't, they look different and i can't, for example, make a "full" block wall on one side and bytes block wall on other
